GNSSLogger/app/src/main/java/com/google/android/apps/location/gps/gnsslogger/SensorFusionListener.java

```diff
@@ -25,7 +25,7 @@
 import android.os.Bundle;
 
 /**  A class representing an interface for logging GPS information. */
-public interface GnssListener {
+public interface SensorFusionListener {
```

Comment:
It sounds like this is a listener for any kind of measurement, whether it be GNSS, Sensor, Fused, or so on- consider renaming it along the lines of `MeasurementListener` and updating the class javadoc to reflect that, i.e. "A class representing an interface for logging a measurement."

Reply:
Also, since we'll now be getting locations from both GNSS and FLP, you'll want to add a second argument to `onLocationChanged` that specifies the source of the location as a string to discern between locations from the two providers, like `onLocationChanged(Location location, String source)`. Then in your callback, you can log the location from FLP with `logger.onLocationChanged(location, "FLP");`, for example.

Reply:
> It sounds like this is a listener for any kind of measurement, whether it be GNSS, Sensor, Fused, or so on- consider renaming it along the lines of `MeasurementListener` and updating the class javadoc to reflect that, i.e. "A class representing an interface for logging a measurement."

Done!

Reply:
> Also, since we'll now be getting locations from both GNSS and FLP, you'll want to add a second argument to `onLocationChanged` that specifies the source of the location as a string to discern between locations from the two providers, like `onLocationChanged(Location location, String source)`. Then in your callback, you can log the location from FLP with `logger.onLocationChanged(location, "FLP");`, for example.

The Android Location class passed in the first argument already has a getProvider() method which is used by the MeasurementListener implementations to access and log this information. Would you still like me to add it as a separate argument? Happy to do so if you'd like!

https://developer.android.com/reference/android/location/Location#getProvider()

Reply:
Oh okay, feel free to leave as is.